### PR TITLE
Update Readme: Add WG github repo & Slack channel

### PR DESCRIPTION
**Pull request by lisaychuang**
Add github repo & slack channel (if avail) for each working group, this should tackle issue #24

### CODE REVIEW

File: working-groups/README.md [hunk 1]
```diff
@@ -8,37 +8,63 @@ AMP’s Working Groups are:
...
 * **Infrastructure.**
   * Responsible for AMP's infrastructure, including building, testing and release.
   * Facilitator: @rsimha
+  * [Infrastructure WG Github repo](https://github.com/ampproject/wg-infra)
+  
 * **Outreach.**
   * Responsible for helping developers who use AMP remain productive and keeping the AMP community healthy.  This includes maintaining and enhancing AMP's developer-facing sites and documentation, maintaining communication channels, organizing community events, etc.
   * Facilitator: @pbakaus
+  * [Outreach WG Github repo](https://github.com/ampproject/wg-outreach)
+  * Slack channel: #wg-outreach
```
Comment (tobie) [inline working-groups/README.md]:
```suggestion
  * Slack channel: [#wg-outreach](https://amphtml.slack.com/messages/wg-outreach)
```

**Comment by lisaychuang** [comment]
@tobie Added links to Slack channels, thanks!

**Comment by mrjoro** [comment]
Thanks @lisaychuang!  Sorry I didn't notice this PR earlier.

I resolved some merge conflicts, and added some more Slack channels to the PR.

I see that you signed the CLA, but @tobie it looks like we don't have a CLA for you and since you helped author the change with your commit we'll need that as well.  Can you [sign one](https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md#contributor-license-agreement) so we can get this change in?

**Comment by mrjoro** [comment]
Nevermind; I see that you are covered by a corp CLA @tobie, and I think @googlebot may just not handle the case of multiple authors in this case correctly.

**Pull request merged**